### PR TITLE
Feature/uh2008 create footer

### LINKED ISSUE #3: UH2008: Página de inicio: footer
# Description

El footer de su maqueta HTML debe ser migrado a react.js. Deberá tener el mismo diseño escogido por el equipo. Se permiten cambios cosméticos sobre el footer pero que no impacten con la estructura de este.

# Criterios de aceptación

Para que esta historia de usuario pueda ser aceptada por el product owner, deberá cumplir con los siguientes criterios de aceptación:
- Se debe visualizar la página de inicio con el footer.
- El footer debe ser responsive

# Notas técnicas

Por favor tener en cuenta las siguientes notas técnicas:
- El footer debe ser un componente reutilizable en react.js. Tener en cuenta que este componente se reuAlizará en todas las páginas del siAo web.
- El equipo puede definir props si así lo considera

### PR DESCRIPTION
**Pull request by Katkat04**
Some photos of the footer 
image
image
image
image

### CODE REVIEW

File: package.json
Comment (Edimar-star) [inline package.json:1]: Well, I see you updated the typescript version, but there's no problem with that anyway.

File: src/components/main/Footer.tsx
Comment (Edimar-star) [inline src/components/main/Footer.tsx:1]: well, I like the structure and it looks good, I don't see anything out of place, I would only say that you have to specify some types of variables (title, options), but it is not mandatory change, it's just to take into account and as the dropdown you created will only be used in the footer, I see no problem in keeping it in the same file, everything in general is fine.

**Review by Edimar-star — APPROVED** [review]
Great job!

**Review by jesusdlacrz — APPROVED** [review]
Excellent, looks very good

**Review by RDCelin — APPROVED** [review]
good job

**Pull request merged**